src/_gettsim/parameters/arbeitsl_geld_2.yaml

```diff
@@ -87,90 +89,145 @@ eink_anr_frei:
     5:
       upper_threshold: inf
       rate_linear: 0
-eink_anr_frei_kinder:
+parameter_anrechnungsfreies_einkommen_mit_kindern_in_bg:
   name:
-    de: Abweichende anrechnungsfreie Einkommensanteile falls Kinder im Haushalt
-    en: null
+    de: >-
+      Parameter der Berechnung des anrechnungsfreien Einkommens wenn Kinder in der
+      Bedarfsgemeinschaft leben
+    en: >-
+      Parameter for calculation of income not subject to transfer withdrawal when
+      children are in the Bedarfsgemeinschaft
   description:
     de: >-
-      Obere Einkommensgrenze bei anzurechnendem Einkommen, Intervall 3, Haushalt mit
-      Kindern. Gilt statt der Parameter in eink_anr_frei für Leistungsbezieher, die mit
-      minderjährigem Kind in BG leben oder die ein minderjähriges Kind haben (zweiteres
-      wird aktuell noch nicht im Code beachtet). § 30 SGB II. Seit 01.04.2011 § 11b SGB
-      II (neugefasst durch B. v. 13.05.2011 BGBl. I S. 850. Artikel 2 G. v. 24.03.2011
-      BGBl. I S. 453).
-      Ersetzt Parameter in eink_anr_frei, falls nicht vorhanden
+      Obere Einkommensgrenze bei anzurechnendem Einkommen, Intervall 3,
+      Bedarfsgemeinschaft mit Kindern. Gilt statt der Parameter in
+      parameter_anrechnungsfreies_einkommen_ohne_kinder_in_bg für Leistungsbezieher, die
+      mit minderjährigem Kind in BG leben oder die ein minderjähriges Kind haben
+      (zweiteres wird aktuell noch nicht im Code beachtet). § 30 SGB II. Seit 01.04.2011
+      § 11b SGB II (neugefasst durch B. v. 13.05.2011 BGBl. I S. 850. Artikel 2 G. v.
+      24.03.2011 BGBl. I S. 453). Ersetzt Parameter in
+      parameter_anrechnungsfreies_einkommen_ohne_kinder_in_bg, falls nicht vorhanden
     en: null
-  type: piecewise_linear
+  type: require_converter
   2005-10-01:
     reference: Artikel 1 G. v. 14.08.2005 BGBl. I S. 2407.
-    deviation_from: arbeitsl_geld_2.eink_anr_frei
+    deviation_from: arbeitsl_geld_2.parameter_anrechnungsfreies_einkommen_ohne_kinder_in_bg
     3:
       upper_threshold: 1500
   2023-07-01:
     reference: G. v. 20.12.2022 BGBl. I S. 2328.
     note: Bürgergeld-Gesetz
-    deviation_from: arbeitsl_geld_2.eink_anr_frei
+    deviation_from: arbeitsl_geld_2.parameter_anrechnungsfreies_einkommen_ohne_kinder_in_bg
     4:
       upper_threshold: 1500
-regelsatz:
+regelsatz_anteilsbasiert:
```

Comment:
@MImmesberger, just a quick heads-up that we should have different parameter names in case the internal structure changes completely over time. Somewhat similar to the `dates_active + leaf_name`, just that we don't have a leaf name here (could mimic through converter functions, though).